config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  devise_for :users, controllers: { registrations: 'registrations' }
+  resources :tweeets
```

Comment:
**REQUIRED**
* It creates the application routes using `resources :posts, only: [:new, :create, :index]` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)).
In our case I think we can replace posts with tweets.

Reply:
done

Reply:
Not done !!!